lib/mysql.js

```diff
@@ -272,6 +272,11 @@ MySQL.prototype.toFields = function (model, data) {
       if (undefined === value) {
         return;
       }
+      // skip property if it cannot be NULL, but `toDatabase` function returns NULL
+      if(props[key].mysql && props[key].mysql.nullable &&
```

Comment:
I intend to normalize the `nullable` to be a boolean. That's why I suggest that you cover `true` too.